src/dry-command-config.ts

```diff
@@ -0,0 +1,103 @@
+import { Cli } from './cli';
+import { CommandProxy } from './command-proxy';
+
+enum DryOption {
+    PackagerCommand = '--dry-packager',
+    KeepPackageJson = '--dry-keep-package-json',
+    SavePackageJsonTo = '--dry-save-package-json-to',
+    LogLevel = '--loglevel',
```

Comment:
Can we extract this logging/verbose feature into another PR?

Reply:
Ok, done